lib/Mojolicious/Plugin/AssetPack/Store.pm

```diff
@@ -257,14 +260,22 @@ sub _download {
 
   return $asset if $attrs{url}->host ne 'local' and $asset = $self->_already_downloaded($url);
 
-  my $tx = $self->ua->get($url);
-  my $h  = $tx->res->headers;
+  my $tx;
+  my $attempts = 1 + $self->retries;
+  while (1) {
+    $tx = $self->ua->get($url);
+    last unless my $err = $tx->error;
+
+    if ($attempts-- > 0) {
```

Comment:
Let's say retires is set to 0. Then `$attempts` is 1. If we'd then end up here we'd still enter this code path and to a retry. Maybe this should have been:
```suggestion
    if (--$attempts > 0) {
```

Reply:
Oops, that was a leftover from an earlier version that used the value in the loop condition.